src/libraries/System.Net.WebSockets/tests/WebSocketDeflateTests.cs

```diff
@@ -434,11 +434,11 @@ public async Task PayloadShouldHaveSimilarSizeWhenSplitIntoSegments(int windowBi
             int frameSize = 2 << windowBits;
 
             byte[] message = new byte[frameSize * 10];
-            Random random = new(0);
+            new Random(0).NextBytes(message);
 
             for (int i = 0; i < message.Length; ++i)
             {
-                message[i] = (byte)random.Next(maxValue: 10);
+                message[i] %= 10;
```

Comment:
Could you explain how you believe this might fix it?

I think it is very unlikely that Random itself is the problem.

Reply:
See my comment here: https://github.com/dotnet/runtime/pull/52052#discussion_r623670733.